packages/amplication-plugin-api/src/pluginVersion/npm-plugin-version.service.ts

```diff
@@ -64,12 +64,12 @@ export class NpmPluginVersionService {
         const npmResponse = await fetch(
           `https://registry.npmjs.org/${pluginNpmName}`
```

Comment:
Unfortunately, there is not REST API with filter by `deprecated` or not packages